client/cypress/integration/event-manager/dataSet.js

```diff
@@ -0,0 +1,21 @@
+var ID = function() {
+  return (
+    '_' +
+    Math.random()
+      .toString(36)
+      .substr(2, 9)
+  );
+};
+export const randomData = {
```

Comment:
@taleymirza please break this down into multiple objects like ValidUser1 ValidUser2 InvalidUser etc etc